src/bournemouth/chat_utils.py

```
@@ -0,0 +1,91 @@
+    finished: bool = False
+
+
+def build_chat_history(message: str, history: list[ChatMessage] | None) -> list[ChatMessage]:
+    """Return chat history with the user message appended."""
+    hist = history or []
+    new_history = [ChatMessage(role=m.role, content=m.content) for m in hist]
+    new_history.append(ChatMessage(role="user", content=message))
+    return new_history
+
+
+async def stream_chat_response(
```

Pylint (3.3.7)

[refactor] Too many arguments (8/5)

(R0913)